### PR TITLE
(ON HOLD) revert: refactor: Migration of the API to async for Meteor 3.0

### PR DESCRIPTION
**Pull request by antobinary**
### What does this PR do?
Reverts #17388 but keeps the changes made later in
#17495 #17496 #17674 
#16828 #17468 #17021 
#17340
#17800

### Closes Issue(s)

Closes # none officially reported

### Motivation
We're seeing some odd behavior in servers. Not sure if this is specifically related but it seemed like a good idea to do the work and have the revert available if needed. Still investigating.

### CODE REVIEW

File: bigbluebutton-html5/imports/api/annotations/addAnnotation.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { check } from 'meteor/check';
-import _ from 'lodash';
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/api/annotations/addAnnotation.js:2]: removing lodash import causes an error `Uncaught (in promise) ReferenceError: _ is not defined on /imports/api/annotations/addAnnotation.js:21`

Screenshot from 2023-05-05 09-30-45

Comment (antobinary) [reply bigbluebutton-html5/imports/api/annotations/addAnnotation.js:2]: Resolved

File: bigbluebutton-html5/imports/api/annotations/server/handlers/whiteboardAnnotations.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import modifyWhiteboardAccess from '/imports/api/whiteboard-multi-user/server/mo
 import clearAnnotations from '../modifiers/clearAnnotations';
 import addAnnotation from '../modifiers/addAnnotation';
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/api/annotations/server/handlers/whiteboardAnnotations.js:5]:
```suggestion
import _ from 'lodash';

```

File: bigbluebutton-html5/imports/api/users/server/eventHandlers.js [hunk 3]
```diff
@@ -8,15 +8,13 @@ import handleEmojiStatus from './handlers/emojiStatus';
 import handleChangeRole from './handlers/changeRole';
 import handleUserPinChanged from './handlers/userPinChanged';
 import handleUserInactivityInspect from './handlers/userInactivityInspect';
-import handleChangeMobileFlag from '/imports/api/users/server/handlers/changeMobileFlag';
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/api/users/server/eventHandlers.js:11]: lines 11 and 19 are part of https://github.com/bigbluebutton/bigbluebutton/pull/17496

File: bigbluebutton-html5/imports/api/users/server/methods/setMobileUser.js [hunk 4]
```diff
@@ -1,27 +1,18 @@
 import { check } from 'meteor/check';
 import Logger from '/imports/startup/server/logger';
+import setMobile from '../modifiers/setMobile';
 import { extractCredentials } from '/imports/api/common/server/helpers';
-import RedisPubSub from '/imports/startup/server/redis';
 
-export default async function setMobileUser() {
+export default function setMobileUser() {
   try {
-    const REDIS_CONFIG = Meteor.settings.private.redis;
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/api/users/server/methods/setMobileUser.js:8]: removed lines are part of https://github.com/bigbluebutton/bigbluebutton/pull/17496

**Comment by Scroody** [comment]
Some of the bugs I found testing this PR:

- Live captions hindering external video progress bar (not really a bug but not good UX)
- External video sharing for more than 2 users sometimes can unsync, so when presenter pauses or skips it doesn't for 1 or more users
- Some bugs related to UNDO tool of TLD that we fixed in the past but apparently is back to hunt us

 Apart from that we have:
- Some slides flickering when we change them
- Auto focusing poll results after clicking them on TLD and then Writing on chat (already reported and issue opened)

**Pull request closed without merging**